package.json

```diff
@@ -13,8 +13,9 @@
         "web"
     ],
     "scripts": {
-        "start": "webpack --mode development --watch",
-        "build": "webpack --mode production",
+        "start": "npm run lint && tsc",
```

Comment:
the start script should run tsc in watch mode (to recompile everytime you do a change) and it should also run lint in parallel. To do this in the admin we use the [concurrently package](https://www.npmjs.com/package/concurrently)